03-MVC-AngularJS/draganddrop/index.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Drag & Drop</title>
+    <link rel="stylesheet" href="styles.css" />
+</head>
+
+<body>
+    <div class="container">
+        <div id="start-text" class="block">
+            <h2>Start Text</h2>
```

Comment:
Rehacer:
La idea de este ejercicio era que hagas drag and drop de un archivo de texto desde tu compu al navegador, esta muy bueno lo que hiciste pero necesito que agregues eso.